### PR TITLE
Fix for #1013, error running sbt test

### PR DESCRIPTION
**Pull request by ekrich**

### CODE REVIEW

File: util/src/main/scala/scala/scalanative/io/VirtualDirectory.scala [hunk 1]
```diff
@@ -98,9 +99,18 @@ object VirtualDirectory {
...
+        Try(FileSystems.newFileSystem(uri, Map("create" -> "false").asJava)) match {
+          case Success(s) => s
+          case Failure(e) =>
+            e match {
+              case e: FileSystemAlreadyExistsException =>
+                FileSystems.getFileSystem(uri)
+              case _ => throw e
+            }
+        }
+      }
```
Comment (densh) [inline util/src/main/scala/scala/scalanative/io/VirtualDirectory.scala]: `scala.uil.Try` might be an overkill here, normal `try catch` would do the job just as well.

**Comment by densh** [comment]
Also #532

**Comment by ekrich** [comment]
It is really strange when tests run in parallel as the output messages get interleaved.

**Comment by densh** [comment]
LGTM, thanks for another nice contribution @ekrich ! 👍

**Pull request merged**